.gitignore

```diff
@@ -8,6 +8,7 @@
 /apache/app-c2corg_ui.wsgi
 /apache/wsgi.conf
 /MANIFEST
+/nbproject
```

Comment:
I see. In this case you should instead configure a global ignore like explained here: https://help.github.com/articles/ignoring-files/.